engine/system/win/sys_local.h

```diff
@@ -50,16 +50,16 @@ class sys_main_c: public sys_IMain {
 	int		KeyToVirtual(byte key);
 	byte	VirtualToKey(int virt);
 
-	HINSTANCE hinst;
-	HICON	icon;
+	HINSTANCE hinst = nullptr;
+	HICON	icon = nullptr;
 
-	class core_IMain* core;
+	class core_IMain* core = nullptr;
 
-	bool	initialised;
-	volatile bool	exitFlag;
-	volatile bool	restartFlag;
-	char*	exitMsg;
-	char*	threadError;
-	bool	errorRaised;
-	int		baseTime;
+	bool	initialised = false;
+	volatile bool	exitFlag = false;
+	volatile bool	restartFlag = false;
+	char*	exitMsg = nullptr;
+	char*	threadError = nullptr;
+	bool	errorRaised = nullptr;

```

Comment:
```suggestion
	bool	errorRaised = false;
```
Error	C2440	'initializing': cannot convert from 'nullptr' to 'bool'